meilisearch-http/src/helpers/tide.rs

```diff
@@ -69,10 +69,10 @@ impl RequestExt for Request<Data> {
         Ok(index)
     }
 
-    fn identifier(&self) -> SResult<String> {
+    fn document_id(&self) -> SResult<String> {
         let name = self
-            .param::<String>("identifier")
-            .map_err(|_| ResponseError::bad_parameter("identifier", "identifier"))?;
+            .param::<String>("documentId")
+            .map_err(|_| ResponseError::bad_parameter("documentId", "primaryKey"))?;
 
```

Comment:
Why is it different than the previous Request? Don't we need to use the primaryKey name used by the user here?

Reply:
It's because in the route the parameter is called `document_id`

e.g. `/indexes/:index/documents/:document_id`